core/loadBalancer/src/whisk/core/loadBalancer/LoadBalancer.scala

```diff
@@ -1,37 +0,0 @@
-/*
- * Copyright 2015-2016 IBM Corporation
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package whisk.core.loadBalancer
-
-import spray.httpx.SprayJsonSupport._
-import spray.json.DefaultJsonProtocol._
-import whisk.core.WhiskConfig
-import whisk.core.WhiskConfig.consulServer
-import whisk.core.WhiskConfig.kafkaHost
-import whisk.core.WhiskConfig.kafkaPartitions
-import whisk.core.WhiskConfig.servicePort
-
-object LoadBalancer {
-
-    def requiredProperties =
-        Map(servicePort -> null) ++
-            kafkaHost ++
-            consulServer ++
-            Map(kafkaPartitions -> null) ++
-            InvokerHealth.requiredProperties
-
```

Comment:
I see where you're coming from, but they haven't been referenced before, so they where safe to nuke. I'll add a companion for the LoadBalancerService